classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
+    super(publish_date, id)
```

Comment:
Kindly, change super method to call only `publish_date`
Reminder of super method in `Item` class : 
```
def initialize(publish_date, archived: false)
    ....
end
```
As you can see above, initialize of `item` object takes 2 parameters, 1 of them is optional and set to `false by default`.

Also, pay attention that you are calling `publish_date` in line 6 while you are not passing the field in `initialize` method, kindly fix that, maybe something like `def initialize(publisher, cover_state, publish_date)`